packages/fastify-dx-vue/virtual/root.vue

```diff
@@ -10,7 +10,15 @@ import Layout from '/dx:layout.vue'
 
 <template>
   <router-view v-slot="{ Component }">
-    <Suspense>
+    <template v-if="$isServer">
```

Comment:
Make this switch in JS, and use 
`import.meta.env.SSR` instead of `isServer` (build only this template in JSX).